pyvista/core/filters/data_set.py

```diff
@@ -393,7 +393,9 @@ def clip_scalar(
         >>> import pyvista as pv
         >>> from pyvista import examples
         >>> dataset = examples.load_hexbeam()
-        >>> _below, _above = dataset.clip_scalar(scalars="sample_point_scalars", value=100, both=True)
```

Comment:
I wonder how the line length limit exactly works. From [the docs](https://blackdoc.readthedocs.io/en/latest/options.html#black) I assume it uses the same line length as `black`. This doesn't sound ideal though, because I we explicitly strive to have shorter lines for docs [as per CONTRIBUTING.md](https://github.com/pyvista/pyvista/blob/main/CONTRIBUTING.rst#coding-style):

> As for docstrings, follow the guidelines specified in [PEP 8 Maximum Line Length](https://www.python.org/dev/peps/pep-0008/#maximum-line-length) of limiting docstrings to 72 characters per line.

Looking at the blackdoc docs it doesn't seem obvious that we can choose a different line length for blackdoc, but [based on the source](https://github.com/keewis/blackdoc/blob/6fa571ad4c905745bf9574bf08248c60529ebe81/blackdoc/blackcompat.py#L106-L109) it seems that the `[blackdoc]` section in `pyproject.toml` takes precedence (as would be logical):
```py
    black_config = pyproject_toml.get("tool", {}).get("black", {})
    blackdoc_config = pyproject_toml.get("tool", {}).get("blackdoc", {})
    config = {**black_config, **blackdoc_config}
```

So do we want to enforce the 72-char length we mention for docstrings?

Reply:
Ha, too late :D

Reply:
Pinging @akaszynski just to make sure we discuss this (and the other, far less important points) before going forward.

Reply:
Hmm, yeah, too bad [the problem seems to be a hornet's nest](https://stackoverflow.com/a/71041192/5067311).

Reply:
Discussed in https://github.com/pyvista/pyvista/pull/4129